llvm/cmake/config-ix.cmake

```diff
@@ -294,6 +294,37 @@ if(LLVM_HAS_LOGF128)
   set(LLVM_HAS_LOGF128 "${HAS_LOGF128}")
 endif()
 
+# Check for ICU.
+if(LLVM_ENABLE_ICU)
```

Comment:
Since the implementation has it such that only one of ICU or iconv is used:
- We should error if both are "forced on".
- If one if "forced on" we should only look for that one, and
- we should otherwise look for ICU first (if enabled) and only look for iconv (if enabled) if ICU is disabled or not found.

Reply:
Thanks, this is done